### ISSUE TITLE
Crashes dedicated server on interaction with Transfer Chest

### ISSUE THREAD

**Issue by Dekamir** [tratteo/TransferChest-1.16#3]
Log and crash report below.
https://gist.github.com/Dekamir/819db756b0049218a701c16aa2a4eaac

Fabric 0.14.21 - 1.19.2 (Modpack: Figuratives)
Windows 10 22H2 x64, OpenJDK Temurin 17.0.5

I can provide more information if needed.

**Comment by tratteo** [comment]
The way entity names are handled, changed from 1.19.2 to 1.19.4, can you please attempt to use this version if you are in 1.19.2 and see if the problem persists?
File: https://legacy.curseforge.com/minecraft/mc-mods/transfer-chest/files/3893759

**Comment by Dekamir** [comment]
CurseForge already uses that file for 1.19.2 profiles. The crash persists.
FYI, 1.19.4 version instantly crashes on 1.19.2 anyway.

Here's a cleaner log.
https://gist.github.com/Dekamir/5ba877d149c6da2b4145aab7b2dbde87

**Comment by tratteo** [comment]
Yes it crashes because there are different ways of handling names of block entities in the two versions.
This seems exactly a problem that I was warned about and fixed with the latest version for 1.19.4. 
Did not receive more warnings after the update, so this is weird. Will try to look into it.
